components/autofill/src/sync/address/mod.rs

```diff
@@ -75,6 +77,9 @@ pub struct AddressPayload {
 #[derive(Default, Deserialize, Serialize)]
 #[serde(default, rename_all = "kebab-case")]
 struct PayloadEntry {
+    pub name: String,
+    // given_name, additional_name and family_name
+    // should exist in PayloadEntry but not in InternalAddress
```

Comment:
maybe expand this comment to explain it exists for "old" clients?

Reply:
updated in https://github.com/mozilla/application-services/pull/6118/commits/340d170c26936bfe2be5a108ae8301ba6f7d7ce9